src/show/pages/public/Home/index.js

```diff
@@ -1,6 +1,26 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
 import { Button, Card, ProgressBar } from '../../../components';
 
 const Home = () => {
+  const [featuredProject, setFeaturedProject] = useState({});
+  const [mostInvested, setMostInvested] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:3000/v1/todays_project/null`)
+      .then(res => setFeaturedProject(res.data))
+      .catch(err => err);
+  }, []);
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:3000/v1/todays_project/`)
+      .then(res => setMostInvested(res.data))
+      .catch(err => err);
+  }, []);
```

Comment:
You have the same array of dependencies for both of your useEffects. Use only one, so fetch data with axios in only one of them.